.github/workflows/pull-request.yml

```
@@ -377,6 +377,7 @@ jobs:
           if [ -f .pidfile_prod ]; then
             kill "$(cat .pidfile_prod)"
           fi
+      - name: Start Development App
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)